src/busctl/busctl.c

```diff
@@ -17,6 +17,7 @@
 #include "capsule-util.h"
 #include "escape.h"
 #include "fd-util.h"
+#include "fdset.h"
```

Comment:
please add "Fixes: #14954" to the commit msg of the first commit.

And we typically prefix the commit title with some string identifying the subsystem this touches, i.e. prefix with `busctl: ` in this case

Reply:
Also please add: "Replaces: #34551" to the first commit.